docs/advanced-topics/docker-alternatives.rst

```diff
@@ -73,6 +73,36 @@ add a line to your ``~/.dockstore/config``:
 
     cromwell-vm-options: -Dconfig.file=<absolute path to your Cromwell conf>
 
+Podman
+---------------
+
+Podman is a daemon-less alternative of Docker that allows end users to run docker/OCI containers without root privileges.
```

Comment:
While we're at it, we should define OCI too. iirc this is the first time they're referenced in our docs.